tests/testthat/test-main.R

```diff
@@ -307,3 +307,41 @@ test_that("list", {
   res$args <- "other"
   expect_error(res$target(res), "orderly bug")
 })
+
+test_that("run: message", {
+  ## Should have no errors
+  path <- prepare_orderly_example("changelog")
+  message <- "[label1] This is a test message."
+  args <- c("--root", path, "run", "--message", message, "example")
+  res <- main_args(args)
+  expect_equal(res$command, "run")
+  expect_equal(res$args, "example")
+  expect_null(res$options$parameters)
+  expect_null(res$options$ref)
+  expect_false(res$options$no_commit)
+  expect_false(res$options$print_log)
+  expect_identical(res$target, main_do_run)
+  expect_equal(res$options$message, message)
+
+  capture.output(res$target(res))
```

Comment:
In the test here, it would be worthwhile checking that the message makes it through to the changelog table I think